scripts/setupVitest.ts

```diff
@@ -65,12 +65,11 @@ expect.extend({
   }
 })
 
-let warn
+let warn = vi.spyOn(console, 'warn')
```

Comment:
This is no longer reassigned, so I suggest switching the `let` to `const`.

Reply:
> This is no longer reassigned, so I suggest switching the `let` to `const`.

updated.